src/BusinessService.ts

```diff
@@ -254,6 +254,30 @@ class BusinessService extends Base {
       subscriptionId: BusinessInvitation["subscriptionId"];
     };
   }
+
+  /**
+   * Update a business invitation by id.
+   *
+   * To accept an invitation, set the `invitation.state` to `accepted`.
+   */
+  async updateInvitation(
+    id: Business["id"] | Business["name"],
```

Comment:
In additional to businessId, we probably need to support business name as url parms, as we only get business name (`company`) here https://github.com/lensapp/lenscloud/blob/main/backend/src/modules/user/user-subscriptions.service.ts#L99

Or, we can pass business id in `getSubscriptionActivationUrl`

Reply:
We have invitationId there as well. This should be enough to get all the related data on API side. Or what do we need also business name?

Invitation already has business and subscriptionId

Reply:
Maybe we need to be more specific and name this acceptBusinessInvitation, as this is the only thing it does?